lib/options.py

```diff
@@ -57,6 +57,14 @@ def __init__(self):
                 help = """List of test suites to look for tests in. Default: "" -
                 means find all available.""")
 
+        parser.add_argument(
+                "--verbose",
```

Comment:
I concerned a bit that the option name is a bit too general then we maybe want. But it is not clear for now, so leave it as is.